test/source/tests/tooling/browser-recipe.ts

```diff
@@ -16,7 +16,29 @@ import { GmailPageRecipe } from '../page-recipe/gmail-page-recipe';
 import { expect } from 'chai';
 import { KeyUtil } from '../../core/crypto/key';
 import { ConfigurationProvider } from '../../mock/lib/api';
+import { MockUserAlias } from '../../mock/google/google-endpoints';
+import { PubKeyLookUpResult } from '../../mock/attester/attester-endpoints';
+import { Dict } from '../../core/common';
+import { GmailMsg } from '../../mock/google/google-data';
+import { somePubkey } from '../../mock/attester/attester-key-constants';
 
+type TestAccount = 'compatibility' | 'compose' | 'ci.tests.gmail';
+interface CommonAcctConfig {
+  google?: {
+    primarySignature?: string;
+    aliases?: MockUserAlias[];
+    draftIdToSave?: string;
+    contacts?: string[];
+    threadNotFoundError?: Record<string, number>;
+    htmlRenderer?: (msgId: string, prerendered?: string) => string | undefined;
+    getMsg?: Dict<Dict<{ error: Error } | { msg: GmailMsg }>>;
+  };
```

Comment:
Can we replace it with just `google?: GoogleConfig`?

Reply:
I thought about it too.
But didn't want to pass `primarySignature: Record<string, string>` and `aliases: Record<string, MockerUserAliases[]?`
This way `google: { primarySignature: flowcryptCompatibilityPrimarySignature },` should be converted to `google: { [acctEmail]: {primarySignature: flowcryptCompatibilityPrimarySignature }},` which seems useless

Reply:
What about this?
```ts
interface CommonAcctConfig {
  google?: GoogleConfig & {
    primarySignature?: string;
    aliases?: MockUserAlias[];
  };
  attester?: AttesterConfig & {
    includeHumanKey?: boolean;
    includeFlowcryptCompatibilityKey?: boolean;
  };
}
```

Reply:
Sounds good!